tests/CMakeLists.txt

```diff
@@ -9,20 +9,26 @@
 # implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 
 CollectSourceFiles(
-  ${CMAKE_CURRENT_SOURCE_DIR}/common
-  COMMON_SOURCES
+  ${CMAKE_CURRENT_SOURCE_DIR}
+  TEST_SOURCES
 )
 
-add_executable(tests-common ${COMMON_SOURCES})
+GroupSources(${CMAKE_CURRENT_SOURCE_DIR})
 
-target_link_libraries(tests-common
+add_executable(tests ${TEST_SOURCES})
+
+set_property(TARGET tests PROPERTY CXX_STANDARD 17)
```

Comment:
Have you tried targetting  trinity-feature-interface ? I don't think we set CXX_STANDARD anywhere else

Reply:
is L24 not enough? `trinity-core-interface` targets `trinity-default-interface` targets `trinity-feature-interface`

Reply:
@Shauren do cmake magic please

Reply:
@Carbenium can you please take a lookt at this to avoid setting CXX_STANDARD in multiple places in the sources ?

![image](https://user-images.githubusercontent.com/1153754/91633627-45bf3780-e9ea-11ea-9749-296d138648d5.png)

It seems Catch2 likes to set its own CXX_STANDARD